### PR TITLE
Use new url quick access for orders list

### LINKED ISSUE #27513: Use the new url for orders list in the quick access links
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

Use the new url for orders list in the quick access links

### Describe the solution you'd like

Change the url by the good one

### Alternatives you've considered

_No response_

### Additional context

Capture d’écran de 2022-01-28 12-32-56

### Do you plan to work on this feature?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

### PR DESCRIPTION
**Pull request by ghost**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop 
| Description?      | Use new url quick access for orders list
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/27513
| How to test?      | Install a shop check the quick url
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27514)

### CODE REVIEW

File: install-dev/data/xml/quick_access.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
   </fields>
   <entities>
     <quick_access id="Orders" new_window="0">
-      <link>index.php?controller=AdminOrders</link>
+      <link>index.php/sell/orders</link>
```
Comment (PululuK) [inline install-dev/data/xml/quick_access.xml:9]: don't forget the upgrade script please?

**Review by PululuK — APPROVED** [review]

File: install-dev/data/xml/quick_access.xml (hunk 1 above)
Comment (ghost) [reply install-dev/data/xml/quick_access.xml:9]: Nice reminder :)

https://github.com/PrestaShop/autoupgrade/pull/457

**Review by matks — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by florine2623** [comment]
Hello @okom3pom ,

Thanks for the PR, it is QA ✅ !

**Comment by Progi1984** [comment]
Thanks @okom3pom & @florine2623

**Pull request merged**